### ISSUE TITLE
Problems of Stitching - Sample Plate Footage

### ISSUE THREAD

**Issue by AndrewHazelden** [making360/making360#206]
@fsoudiere @vickixhuang  I was wondering what your interest would be in starting an official collection of `making 360`  creative commons licensed plate footage resources that could be used by people who are interested in following along with the exercises in the "Problems of Stitching" chapters. 

Having several short sample clips and stills from the different types of live action panoramic video rigs would let people try their hand a 360&deg; post production workflow techniques. This could help students, new filmmakers, and visual effects artists see what works and what doesn't when trying to  choose the right rig, shooting style, and what it is like to stitch real world footage shot in a variety of indoor, outdoor, or night time settings.

**Comment by fsoudiere** [comment]
Totally agree! Lets do it! I think the best would be for us to update the site making360 soon and have all resources there. But, in meantime, I think we could add a page to the book that list few things such as sample footage, best 360 reads, cheat sheets, log sheets,... and maybe a download icon next to each chapter pointing to the sample footage. Where should we host that footage? Dropbox? Amazon? Godaddy?

**Comment by AndrewHazelden** [comment]
Amazon S3 would probably be the best way to host the large media files cheaply with high availability. Dropbox would limit the downloads after many people try to access it in a short period of time.

**Comment by AndrewHazelden** [comment]
I heard that GitHub is working on adding [Git Large File Storage (LFS)](https://github.com/blog/1986-announcing-git-large-file-storage-lfs) support which sounds interesting too.

**Comment by AndrewHazelden** [comment]
Also another helpful resource would be to come up with an improved set of storyboarding templates to help people plan their 360&deg; shots with indications of the horizon line and other framing details. Coming up with a few different layouts for the LatLong projection with stitch line centric safety zone framing, and a front viewable area FOV zone indicator would be really helpful.

Last year I created a simple [fulldome centric storyboard template](http://www.andrewhazelden.com/blog/2014/01/creating-fulldome-storyboards/) in PDF format on my blog.

**Comment by fsoudiere** [comment]
Yes I am working hard this one. We even printed compass to draw storyboards as 360 space from birds eye view, with different marks to pick your angles.

**Comment by fsoudiere** [comment]
@AndrewHazelden https://github.com/making360/making360/pull/207

**Comment by fsoudiere** [comment]
if you want to upload or others want to upload i need to first get some sponsors lol to pay for this, and also figure out the users granting policies from S3... need to think this through but i like for the digital copy of the book to have a small icon to download resources from the book. then also it will be on the site.

**Comment by AndrewHazelden** [comment]
@fsoudiere I sent along a small PayPal donation to your fabiensoudiere@gmail.com address to help kick off the initial setup of the Amazon S3 account. Once some media has been assembled I'm sure more people will come forward to help support the hosting costs too.

I could see a resource like this becoming the panoramic stitching equivalent of the classic Cineon film scanner [Kodak Digital LAD Test Image](http://motion.kodak.com/motion/Support/Technical_Information/Lab_Tools_And_Techniques/digitallad.htm).

**Comment by nuclearsugar** [comment]
I'd be willing to shoot some 360 footage with my 10 camera rig. Both still and moving shots.

**Comment by fsoudiere** [comment]
@nuclearsugar genius idea haha! then @AndrewHazelden have you every dealt with user privileges on Amazon S3?

**Comment by AndrewHazelden** [comment]
@fsoudiere I've only used S3 with a single administrative account for the Access Key ID. 
You can use the Amazon S3 Web based Management tools for setting up the folders and their permissions. 

The free [Cyberduck](https://cyberduck.io/) file transfer tool is how I typically upload media files to S3 from a Windows or Mac system using a drag and drop approach. You can also use the web based Amazon S3 Management console to upload media too.

**Comment by AlexPearceVR** [comment]
Wonderful idea @AndrewHazelden! I have some footage from 7cam 360 heros, 2 and 3 cam iZugar rigs and I can ask some of my clients if I can use some of their footage from 14cam 3d rig, freedom 360, Theta S etc. I'll be checking in with @fsoudiere on what the protocol is moving forward.

**Comment by fsoudiere** [comment]
@making360/m360  
We just signed up for a making360 Amazon S3 server. Contact me directly fab@making360.com so I can grant you permissions to upload content to our server.

We added a download icon near the chapter title when sample is available.

<img width="442" alt="screen shot 2015-12-07 at 1 04 23 pm" src="https://cloud.githubusercontent.com/assets/1613933/11639764/95b81d1e-9ce3-11e5-8767-285d828bc822.png">

**Comment by nuclearsugar** [comment]
I can record some 360 footage later this week. Requests? Here is my plan so far.
--- Moving shot using Indie Dolly
--- Moving shot using Motorized wheelchair shot
--- Moving shot using Manual wheelchair shot
--- A few still shots
--- Shot with noisy footage

To help keep bandwidth usage under control, it might be wise to agree on a max limit for uploaded footage. For example when shooting at 2.7k 30fps, a 20 second shot is about 165MB. So with my 10 camera rig that comes out to on average 1.5GB per 30 seconds.

**Comment by fsoudiere** [comment]
@nuclearsugar @AndrewHazelden just added you guys to the Amazon console.

**Comment by AndrewHazelden** [comment]
Thanks @fsoudiere! I'm just testing the m360 S3 account out now by downloading chap30.zip.

**Comment by nuclearsugar** [comment]
@fsoudiere So I'm going to keep each of my shots around 1 minute, so that I can fit the maximum size of 4GB per ZIP (for Amazon upload).

I have a bit of work to do in shooting, and I don't want my work to be in vain since uploaded footage must be associated with a specific chapter. So can we confirm/brainstorm that shots that I'll be contributing?

--- Moving shot using Indie Dolly = stabilization chapter
--- Moving shot using Motorized wheelchair shot = stabilization chapter
--- Moving shot using Manual wheelchair shot = stabilization chapter
--- A few still shots: indoors, outdoors = what chapters?
--- Noisy footage = already done?

**Comment by fsoudiere** [comment]
I would say lets have one main zip to download per chapter. So lets have one for Stabilization, and one for Moving Shots in stitching. However you can still upload all footage to the server, and I will think about how to best implement multiple sample footage per chapter. We may need an interface for people to choose footage that want to download.

then do Noisy Footage, you may need to set the wrong iso, ev comp to really make it noisy.

see if you can do some basic chapters like Control Points and Masking Markers.

**Comment by fsoudiere** [comment]
@nuclearsugar @AndrewHazelden @AlexPearceVR Hey guys, do you have any footage we can add to the amazon cdn? I may add some later but if you did upload stuff please provide the link in the chapters or here so I can do it.

**Comment by nuclearsugar** [comment]
Now that I'm back in town and done with my holiday travels, I'll try to dedicate some time to recording some open source 360 video.

**Comment by fsoudiere** [comment]
Would love to maybe do a new release of the pdf when we have at least one sample footage per stitching chapter. new releases will prob be the translations.

**Comment by nuclearsugar** [comment]
I was upgrading the firmware on my GoPro cameras recently and one of them bricked. So I'm unable to shoot a full 360 currently. Will try to get a new one soon.

But having one sample footage per stitching chapter seems a bit unnecessary. You might want to list the stitching chapters and then describe the kind of example shots you need. Then ask people to sign up for shooting specific shots.

**Comment by fsoudiere** [comment]
@making360/m360 just uploaded about 10 samples test footage for the book. not the best but will be helpful to most newcomers. some needed footage would include: 

- footage from diff rigs (z2x, z3x, z4x, f360, abyss, jump, jaunt, xiaomi, entaniya, 360heros) for chapter: Platonic rigs. 
- footage for chapter Stabilization
- footage for chapter Lighting
- footage for chapter Color Matching
- footage for chapter Motion Flash (reference signals)
- footage for chapter Stereoscopic 3d 360
- footage for chapter First Person (multiple solutions)
- drone footage
-green screen footage

no rush but you guys should upload whenever possible. I will try to make some changes to few chapters and create a new release first week of february 2016. I will also send out a newsletter to announce the sample footage and a potential pre-order for the next batch of hardcopies. 

have a good weekend!

fab

**Comment by Arkasarkas** [comment]
Hey guys, would love to join/contribute tot his project. I'm currently testing the Z2X template and trying some footage coming from my dslr with 8mm fisheye. If this is not the right place, please let me know. And big up Fabien for the youtube videos and the PDF, it's been a great help thusfar!
